src/ILLink.RoslynAnalyzer/RequiresUnreferencedCodeAnalyzer.cs

```diff
@@ -31,6 +33,49 @@ public sealed class RequiresUnreferencedCodeAnalyzer : RequiresAnalyzerBase
 				operationContext.Operation.Syntax.GetLocation ()));
 		};
 
+		[SuppressMessage ("MicrosoftCodeAnalysisPerformance", "RS1008",
+			Justification = "This action is registered through a compilation start action, so that the instances which " +
+			"can register this operation action will not outlive a compilation's lifetime, avoiding the possibility of " +
+			"this data causing stale compilations to remain in memory.")]
```

Comment:
Can you please explain why this action causes the warning and the one above doesn't? I don't see how they're really different...